src/style.css

```diff
@@ -0,0 +1,84 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.block {
+  flex-grow: 1;
+  height: 300px;
+  width: 300px;
+  background-color: rgb(255, 0, 0);
+  text-align: center;
+  line-height: 300px;
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .block-3 {
+    order: 4;
+  }
+
+  .block-4 {
+    order: 3;
+  }
+
+  .block-5 {
+    order: 5;
+  }
+
+  .block-6 {
+    order: 6;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    flex-basis: 33.33%;
```

Comment:
try `calc(100% / 3)`

Reply:
Okay thank you. tried to divide without calc(). So to calculate in CSS, do we always use the calc() function or it's only in this case?

Reply:
if you need 50%, then no need to use calc. 50% is exactly the same as calc(100% / 2). calc is performed by the browser and if you have too many of them, the performance might drop.
but in this case, there is no way to set 33.3333.....% without calc, so we should use it

Reply:
@MabuzaM

Reply:
Thank you @ihor-jpeg understood